pydrive2/files.py

```diff
@@ -355,7 +355,7 @@ def download(fd, request):
             # But that would first require a slow call to FetchMetadata().
             # We prefer to try-except for speed.
             try:
-                download(fd, files.get_media(fileId=file_id))
+                download(fd, files.get_media(fileId=file_id, acknowledgeAbuse=True))
```

Comment:
Hi @pengyunie !

Nice catch! :rocket: 

Seems like it would make sense to add it as an option for GetContentFile itself, so we could then use it in dvc.

Also thanks for pointing out that line in the docs about

```
Your application should interactively warn the user before using this query parameter.
```

I wonder if this is something that we are obliged to do (or our token will get rewoked or something). And if we really have to do it interactively or if an explicit config option (in dvc config) is enough. :thinking: CC @shcheklein

Reply:
Good catch @pengyunie indeed! 

I agree with @efiop - it should be an option in some way on the PyDrive2 API level. We should not keep it "always" enabled. PyDrive2 is a general library, DVC is not the only client.

I also agree that it should be a config option on the DVC end.

> Your application should interactively warn the user before using this query parameter.

Good question. It seems we have to write a message at least if it's enabled. I would check some other sync/CLI tools- how do they handle this.

Reply:
rclone has a cli option, config option and an env var: https://rclone.org/drive/ , so we might get away with a dvc config option (disabled by default) and warn the user properly when we run into this during dvc operation.